kimetaro.py

```diff
@@ -50,17 +60,22 @@
         return
 
     if message.content.startswith('/hey' + COMMAND_SUFFIX):
-        await send_reply(message, 'おーきに')
+        await send_reply(message, random.choice(
+            kimetaro.parser.getParameter("HEY_MESSAGE1")[0])[1:-1])
 
     if message.content.startswith('/kimetaro' + COMMAND_SUFFIX):
-        await send_reply(message, 'よーうし、決めたるで〜')
-        await send_reply(message, 'むむっ、これや！\n')
+        await send_reply(message, random.choice(
+            kimetaro.parser.getParameter("KIMETARO_MESSAGE1")[0])[1:-1])
+        await send_reply(message, random.choice(
```

Comment:
Just confirming.
Can we combine these two reply?

Reply:
Yes, we can combine these two reply.
However, because there is a limit to the number of messages that can be returned at one time, it is may not better that combine them.

Reply:
`number of messages` means message length?

Reply:
Yes. It may be contain 200, 300  or below characters at one time.
Depending on the messages chosen at random, the combination may exceed the length of the characters.